the-collab-lab/tcl-14-smart-shopping-list
# Mja lb issue seven

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



## Related Issue



## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After




## Testing Steps / QA Criteria



The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item
![Captura](https://user-images.githubusercontent.com/38761773/97741128-48a0dc00-1ab0-11eb-9f63-9ed21ecc5a69.PNG)
